modules/services/compose/src/main/java/au/com/shiftyjelly/pocketcasts/compose/components/GradientIcon.kt

```diff
@@ -56,6 +58,43 @@ fun GradientIcon(
     )
 }
 
+@Composable
+fun GradientIcon(
+    painter: Painter,
+    gradientBrush: Brush,
+    contentDescription: String? = null,
+    blendMode: BlendMode = BlendMode.SrcAtop,
```

Comment:
Should the `blendMode` be customizable if this is meant to be a gradient icon?

Reply:
I searched for gradient options and I found these `SrcAtop`, `DstAtop`, `SrcIn` and `DstIn`, so I thought it could be a good idea to give the option to change the blendMode

Reply:
In that case I'd like to suggest to rename the component to `BrushIcon` as it doesn't need to be a gradient `Brush` but any `Brush` can be used.